verifly.gemspec

```diff
@@ -6,20 +6,29 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'verifly/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = 'verifly'
-  spec.version       = Verifly::VERSION
-  spec.authors       = ['Alexander Smirnov']
-  spec.email         = ['begdory4@gmail.com']
+  spec.name = 'verifly'
+  spec.version = Verifly::VERSION
+  spec.authors = ['Alexander Smirnov']
+  spec.email = ['begdory4@gmail.com']
 
-  spec.summary       = ''
-  spec.description   = ''
-  spec.homepage      = 'http://example.com'
+  spec.summary = <<~SUMMARY.gsub(/\s+/, '')
+    An api to run sequential checks like 'ActiveModel::Validations'
+    do, but with generic messages instead of errors
+  SUMMARY
 
-  spec.files         = Dir['lib/**/*.rb']
-  spec.executables   = []
+  spec.description = <<~DESCRIPTION.gsub(/\s+/, '')
+    See more info at
+    http://www.rubydoc.info/gems/verifly/#{Verifly::VERSION}
+  DESCRIPTION
+  spec.homepage = 'https://github.com/umbrellio/verifly'
+  spec.license = 'MIT'
+
+  spec.files = Dir['lib/**/*.rb']
+  spec.executables = []
   spec.require_paths = %w[lib]
 
-  spec.add_development_dependency 'pry'
+  spec.required_ruby_version = '~> 2.3'
+  spec.add_development_dependency 'pry', '~> 0.10'
```

Comment:
Because it used in development? It required in bin/console and spec/spec_helper.rb